### PR TITLE
update labels

### PR DESCRIPTION
**Pull request by afeld**
This pull request does a number of things:

- Switches to the new Initiative names
   - Renames some:
      - `acquisition` → `SaaS`
      - `infrastructure improvement` → `infrastructure`
      - `internal workflow` → `internal team`
      - `mac` → `customer support`
      - `software authorizations` → `custom software`
   - Deprecates some, which will be manually merged into the remaining ones and then deleted
- Adds descriptions to each label, which you can see on [the labels page](https://github.com/18F/tts-tech-portfolio/labels)
   - These appear when you hover over labels in issues or the project board as well
   - I just took a first pass; easy to change those after the fact
- Adds a script to help with renaming labels

### CODE REVIEW

File: terraform/README.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
       `~/.aws/config:`
 
-      ```hcl
-         ~/.aws/config:
-         [profile_name]
-         region=us-west-2
+      ```ini
+      [profile_name]
+      region=us-west-2
       ```
```
Comment (afeld) [inline terraform/README.md:22]: Everything above is minor cleanup; easiest to review ignoring whitespace.

File: terraform/repo/vars.tf [hunk 2]
```diff
@@ -2,30 +2,114 @@ variable "repo" {
...
+      description = "Relating to process or training within the Tech Portfolio, or communication about the team outwards"
+    }
+    "i: misc" = {
+      color       = local.label_colors.initiatives
+      description = "Issues that don't fit in another Initiative"
+    }
+    "i: SaaS" = {
+      color       = local.label_colors.initiatives
+      description = "Relating to third-party software: acquisition, security compliance, etc."
+    }
```
Comment (afeld) [inline terraform/repo/vars.tf:64]: New Initiative names are above.

Comment (JJediny) [reply terraform/repo/vars.tf:64]: per slack thread lg

File: terraform/README.md [hunk 3]
```diff
@@ -42,3 +41,15 @@ To `plan`/`apply` changes:
...
+
+1. Edit the list in [`repo/vars.tf`](repo/vars.tf).
+1. Get the migration commands.
+
+   ```sh
+   python rename_label.py "<label before>" "<label after>"
+   ```
+
+1. Note this doesn't actually perform the migration. Copy and paste the output into the terminal to execute.
+1. When running `terraform plan`/`apply`, it should show those labels being changed, not removed+added.
```
Comment (JJediny) [inline terraform/README.md:55]: Assuming this is one time or in need during conversion from 1:1 ok

**Review by JJediny — APPROVED** [review]
LG not sure the advantage of moving from vars to local but will defer all else looks fine

**Comment by afeld** [comment]
> not sure the advantage of moving from vars to local

Probably over-engineered, but I wanted to specify the colors in a single place, and vars can't refer to other vars.

**Pull request merged**